rules-tests/CodingStyle/Rector/Namespace_/ImportFullyQualifiedNamesRector/Fixture/skip_same_namespaced_used_class.php.inc

```diff
@@ -11,18 +11,3 @@ class SkipSameNamespacedUsedClass
     {
     }
 }
------
-<?php
-
-namespace Rector\Tests\CodingStyle\Rector\Namespace_\ImportFullyQualifiedNamesRector\Fixture;
-
-use Rector\Tests\CodingStyle\Rector\Namespace_\ImportFullyQualifiedNamesRector\Source\SharedShortName;
-class SkipSameNamespacedUsedClass
-{
-    /**
-     * @return SharedShortName
-     */
-    public function run(): SharedShortName
-    {
-    }
-}
```

Comment:
this expected output seems previously correct, as the docblock `@return` changed to short name, the "skip" here means skipping from re-import